esm/esmfold/v1/esmfold.py

```diff
@@ -58,6 +75,13 @@ def __init__(self, esmfold_config=None, **kwargs):
             nn.ReLU(),
             nn.Linear(c_s, c_s),
         )
+        if cfg.use_esm_attn_map:
```

Comment:
Will it work without adding the respective field to ESMFoldConfig?

Reply:
They should be properly added.